integration/scripts/src/runTest.ts

```diff
@@ -0,0 +1,76 @@
+import { ChainlinkNode } from './chainlinkNode'
+import { fetchTests } from './tests'
+import { fetchArgs, fetchCredentials } from './common'
+import * as assert from './asserts'
+
+const main = async () => {
+  const tests = fetchTests()
+
+  const { chainlinkUrl } = fetchArgs()
+  const credentials = fetchCredentials()
+  const node = new ChainlinkNode(chainlinkUrl, credentials)
+
+  const ctx = await assert.context(async (ctx) => {
+    for (let i = 0; i < tests.length; i++) {
+      const test = tests[i]
+      await assert.newTest(test, async () => {
+        const jobCount = (await node.getJobs()).meta?.count || 0
+        let jobId: string
+        await assert.it('creates job', ctx, async () => {
+          jobId = await addJob(node, test.params)
+          assert.isFalse(test, !jobId, 'got a job ID')
+          const newJobCount = (await node.getJobs()).meta?.count
+          assert.equals(test, newJobCount, jobCount + 1, 'job count should increase by 1')
```

Comment:
Could this work?
```js
assert.equals(test, jobCount, jobCount + 1, 'job count should increase by 1')
```

Reply:
This wouldn't work, as we need to check the job count on the CL node.